tests/integration/cmd_push_test.go

```diff
@@ -55,27 +55,27 @@ var _ = Describe("odo push command tests", func() {
 		})
 
 		It("should be able to create a file, push, delete, then push again propagating the deletions", func() {
-			helper.CmdShouldPass("git", "clone", "https://github.com/openshift/ruby-ex", context+"/ruby-ex")
-			helper.CmdShouldPass("odo", "component", "create", "ruby", cmpName, "--project", project, "--context", context+"/ruby-ex", "--app", appName)
+			helper.CmdShouldPass("git", "clone", "https://github.com/openshift/nodejs-ex", context+"/nodejs-ex")
```

Comment:
Why are we changing it from ruby to nodejs? Is it because we don't "officially" support ruby (yet?)

Reply:
Yes and some variables like `DEPLOYMENT_DIR` are not populated in ruby s2i images